pandas/core/frame.py

```diff
@@ -1598,8 +1597,9 @@ def to_feather(self, fname):
         from pandas.io.feather_format import to_feather
         to_feather(self, fname)
 
-    @Substitution(header='Write out column names. If a list of string is given, \
-it is assumed to be aliases for the column names')
+    @Substitution(header='Write out the column names. If a list of strings '
+                         'is given, it is assumed to be aliases for the '
+                         'column names')
```

Comment:
Already copied the edits into the other docstrings, choosing bed over _shared_docs.